proofwatch/proofwatch.go

```diff
@@ -7,30 +7,47 @@ import (
 	"log"
 	"time"
 
+	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
 	olog "go.opentelemetry.io/otel/log"
 	"go.opentelemetry.io/otel/log/global"
 	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	// ScopeName is the instrumentation scope name.
+	ScopeName = "github.com/complytime/complybeacon/proofwatch"
 )
 
 type ProofWatch struct {
-	name          string
 	logger        olog.Logger
+	tracer        trace.Tracer
 	observer      *EvidenceObserver
 	levelSeverity olog.Severity
 }
 
 // NewProofWatch creates a new ProofWatch instance with OpenTelemetry logging.
-func NewProofWatch(name string, meter metric.Meter) (*ProofWatch, error) {
+func NewProofWatch(opts ...OptionFunc) (*ProofWatch, error) {
+	cfg := config{
+		MeterProvider:  otel.GetMeterProvider(),
+		LoggerProvider: global.GetLoggerProvider(),
+		TracerProvider: otel.GetTracerProvider(),
+	}
+	for _, opt := range opts {
+		opt(&cfg)
+	}
+
+	meter := cfg.MeterProvider.Meter(ScopeName, metric.WithInstrumentationVersion(Version()))
 	observer, err := NewEvidenceObserver(meter)
```

Comment:
@jpower432 should this be taking a `metric.MeterProvider` instead of a `Meter`?  Not sure if this guideline statement applies here? "All instrumentation packages MUST NOT provide an option to accept a Tracer or Meter."